Libs/Core/ctkJobScheduler.cpp

```diff
@@ -291,7 +307,7 @@ void ctkJobScheduler::stopAllJobs(bool stopPersistentJobs)
       }
 
     job->setStatus(ctkAbstractJob::JobStatus::Stopped);
-    this->deleteJob(job->jobUID());
+    job->canceled();
```

Comment:
Method name should be a verb, such as `job->cancel()`, or `job->setCanceled(true)`.

Reply:
it is actually a signal. the ctkJobScheduler oberse taht signal and it will destroy the job object. Please note that in that case, the wroker has not beed started yet and the job is still in the main thead (see comment just few lines before)